### DISCUSSION TITLE
Modding code to move CS, Reset, CD onto I2C

### DISCUSSION THREAD

**Discussion by GeeKoC** [Bodmer/TFT_eSPI#877] [New processor/display support or capability request] [answered]
Hi all
My discussion is about how many files I would need to change to create a compiler switch to move the lower speed signal lines onto an I2C 8 or 16 bit device.  idea is to reserve as many eps32-cam pins whilst extending the port type and count via I2C. And Or. Creating a way to have even more spi devices share the one spi subsystem (Esp32 HSPI or VSPI) because the control lines to the spi chips/modules run thru I2C.
Does anyone have an understanding they can share about the files to change?
Will try the basic text search for CS, RESET and DC.
There already are compiler switches for ESP32, ESP8266, bluepill.
Looking to now compiler switches for MCP32008, MCP32017 PCF....

**Comment by Bodmer** [comment, marked as answer]
Have a look at #495
